templates/dashboard.html

```diff
@@ -23,6 +33,24 @@ <h5 class="card-title">Master account WAVES balance</h5>
             </div>
         </div>
 
+        <div class="card">
```

Comment:
same here, merge this with the card above "Master account WAVES balance"

Reply:
moved as well.